test/MyFirstApiConsume.test.js

```diff
@@ -0,0 +1,78 @@
+const agent = require('superagent');
+const statusCode = require('http-status-codes');
+const chai = require('chai');
+
+const expect = chai.expect;
+
+describe('First Api Tests', () => {
+
+    it('Consume GET Service', async () => {
+        const response = await agent.get('https://httpbin.org/ip');
+      
+        expect(response.status).to.equal(statusCode.OK);
```

Comment:
For the next ones I think it's better to use satusCode.StatusCodes.OK, this one is deprecated.